app/code/Magento/Backend/view/adminhtml/layout/default.xml

```diff
@@ -17,6 +17,7 @@
     <body>
         <attribute name="id" value="html-body"/>
         <block name="require.js" class="Magento\Backend\Block\Page\RequireJs" template="Magento_Backend::page/js/require_js.phtml"/>
+        <block class="Magento\Framework\View\Element\Text\ListText" name="head.additional"/>
```

Comment:
Maybe it's better to use `Magento\Backend\Block\Text\ListText` ? as this is adminhtml area ?

Reply:
What's the added value of using that more-specific class over this one? I don't see that it adds any value, and if it gets deprecated (probably for that same reason) then it would be a BiC to go back to the parent version – while the opposite (going from the parent version to the child version) would be an easier change because it's not a BiC.

Reply:
To be consistent with namespaces

Reply:
and i don't see any reason for what this class can be deprecated, and this class defined in xml, so we can easily change to another one.

Reply:
> this class defined in xml, so we can easily change to another one.
That's exactly my point: if we leave it as-is (using the framework type), then making changes in the future would be even easier than if we changed it to the backend type.

Being consistent with namespaces is a good principle when you're designing your type hierarchy. But it's less important when you're using it, especially if other principles like ISP could be more relevant.

The reasons why I'd leave it as-is:
1. ISP recommends using only the most meaningful contract, and nothing "more" than that.
2. Since we're using the more meaningful contract, which also happens to be the most generic type, it's also easier to make changes without causing BC breaks and there's more room for LSP (even though in practice it will never be applied at that level).
3. The framework type is the only type that carries actual meaning. The existence of the backend type that adds no value is questionable in the first place.

Having said that, I don't mind making the change if Magento really needs it to merge this code 🤷‍♂️. Are you in charge of making the final call on this PR or is this just a suggestion?

Reply:
@nazar65 in this case I do agree with @aredridel - we don’t really have any reason to use there class from backend module as it does the same. I would prefer keeping it as is

Reply:
... Hm? With what?

Reply:
> ... Hm? With what?

With `Magento\Framework\View\Element\Text\ListText`

Reply:
it was just suggestion, so if everyone  agree with `Magento\Framework\View\Element\Text\ListText` so just ignore my suggection